### DISCUSSION TITLE
Embed Console Apps in a Window

### DISCUSSION THREAD

**Discussion by BangCtrl** [AvaloniaUI/Avalonia#8477] [Q&A]
Is it possible to embed console apps into a window? I need to do something like issue #8012.

**Comment by kekekeks** [comment]
https://github.com/darrenstarr/VtNetCore
https://github.com/VitalElement/VtNetCore.Avalonia

**Comment by BangCtrl** [comment]
I also found this - https://github.com/VitalElement/AvalonStudio.TerminalEmulator.

**Reply by BangCtrl** [reply]
Got it working.

**Reply by BangCtrl** [reply]
I'm starting to see the downsides.
